package/upgrade/lib.sh

```diff
@@ -14,7 +14,13 @@ download_file()
   local output=$2
 
   echo "Downloading the file from \"$url\" to \"$output\"..."
-  curl -sSfL --retry 10 --retry-connrefused "$url" -o "$output"
+  local i=0
+  while [[ "$i" -lt 100 ]]; do
```

Comment:
In previous PR, retry times is 10. Why change to 100 in this PR?

Reply:
Because the retry algorithm of curl is exponential backoff by default, the accumulated time of 10 retries is about 1023 (1+2+4+8+...+512) seconds. 1000 (10*100) seconds is nearly the same.

Reply:
OK, you want the retry interval is the same as previous PR.